README.md

```diff
@@ -1,6 +1,6 @@
 # @hirez_io/observer-spy 👀💪
 
-A simple little class and a helper function that help making Observable testing a breeze
+A simple little class and a helper function that helps make Observable testing a breeze
```

Comment:
Oh, I thought that because it refers to both the class and the function those form a group and hence the "help".
Please correct me if I'm wrong